e2e/Makefile

```diff
@@ -54,5 +54,10 @@ help: ## Show this help
 ### Artifacts
 ###
 
-$(bats):
+$(bats): package-lock.json
 	@npm install
+
+package-lock.json: package.json
+	@npm install
+
+package.json:
```

Comment:
This causes npm to be installed twice per `make e2e-test` run. Is that intended?
I would expect that it only installs packages when bats isn't installed yet.

```console
$ make e2e-test
make[1]: Entering directory '/home/chris/Documents/source/vshn/k8up/e2e'

up to date, audited 2 packages in 576ms

found 0 vulnerabilities
make[1]: Leaving directory '/home/chris/Documents/source/vshn/k8up/e2e'
go build -o /home/chris/Documents/source/vshn/k8up/./k8up main.go
...
make[1]: Entering directory '/home/chris/Documents/source/vshn/k8up/e2e'

up to date, audited 2 packages in 584ms

found 0 vulnerabilities
 ✓ lint assertions 
 ✓ verify the deployment 
 ✓ verify a backup 

3 tests, 0 failures
```

Reply:
🤔

I've deleted it for now